Character_Builder/models.py

```diff
@@ -53,14 +92,13 @@ class Character(models.Model):
     # This method returns a string that represents this class.
     # Similar to toString() from java
     def __str__(self):
-        return (self.characterID, self.characterName)
+        return self.characterName
```

Comment:
I understand that it may be a good idea to just return a character name for a string that represents the class, but is there ever a time where two characters may have the same name?

Or is that a separate function to return additional information about a given class?